bin/overview-modules/0_overview-heat-flow-module.ipynb

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
<blockquote>An introduction to <strong>the </strong>conductive heat transfer mechanism within the Earth. Add bolded word (<strong>the</strong>).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>